components/follow_button/view.coffee

```diff
@@ -52,7 +51,7 @@ module.exports = class FollowButton extends Backbone.View
 
     unless @following
       mediator.trigger 'clickFollowButton'
-      return unless @authOnClick
+      return if NEW_ARTIST_PAGE_CTA is 'new_cta'
```

Comment:
When we're in the new CTA group, we want to change the follow button's behavior to fire this mediator event, which the CTA bar is listening to.

I just realized we also only want this to happen when we're on an artist page (doh!), will update.